tm2/pkg/std/memfile.go

```diff
@@ -90,6 +90,8 @@ func (mempkg *MemPackage) Validate() error {
 	return nil
 }
 
+const licenseName = "LICENSE"
```

Comment:
Aren't documents like `LICENSE.md` or `LICENSE.txt` also valid license documents, regardless of their file extension?
Why do we force them to be named exclusively `LICENSE`?

Reply:
This definitely warrants a larger discussion, but it is like this currently because this is the "allowed" filename for licenses:
https://github.com/gnolang/gno/blob/31a5f2ea63ab1f069201bfa7ad35e9eada8ec6d9/gnovm/pkg/gnolang/nodes.go#L1110-L1114

There are some other parts to this that seems strange and need to be better considered so I created an issue:
https://github.com/gnolang/gno/issues/2421

The particular issue that this PR addresses will unblock me on something; I can fill you in on the details.